File: Laravel/app/Biab/Resources/Reviews.php
<?php

namespace App\Biab\Resources;

use App\Biab\Client;

/** The org-wide review wall (per-product reviews live on Storefront). */
class Reviews
{
    public function __construct(private readonly Client $client)
    {
    }

    /** @return array<string, mixed> */
    public function list(?int $limit = null, ?int $offset = null): array
    {
        return $this->client->get('reviews', [
            'limit' => $limit,
            'offset' => $offset,
        ]);
    }
}

File: Phoenix/lib/biab_starter/biab/resources/reviews.ex
defmodule BiabStarter.Biab.Resources.Reviews do
  @moduledoc "The org-wide review wall. Per-product reviews live on Storefront."

  alias BiabStarter.Biab.Client

  def list(client, limit \\ 10, offset \\ 0),
    do: Client.get(client, "reviews", limit: limit, offset: offset)
end
